frontend/appflowy_flutter/lib/plugins/database/board/presentation/toolbar/board_setting_bar.dart

```diff
@@ -38,6 +43,9 @@ class BoardSettingBar extends StatelessWidget {
                 FilterButton(
                   toggleExtension: toggleExtension,
                 ),
+                if (isReference) const HSpace(2),
+                if (isReference)
+                  ViewDatabaseButton(view: databaseController.view),
```

Comment:
Using `...[const HSpace(2), ViewDatabaseButton(view: databaseController.view)]` would be better.